### ISSUE TITLE
Vertical alignment is not managed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#2]
```
Vertical alignment is not managed.
```

Original issue reported on code.google.com by `jerome.m...@gmail.com` on 9 Mar 2011 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
OK I will see this problem.

Many Thanks Jerome to report this bug.
```

Original comment by `angelo.z...@gmail.com` on 14 Apr 2011 at 9:47
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the 1.0.0. See JUnit at 
http://code.google.com/p/xdocreport/source/browse/thirdparties-extension/org.apa
che.poi.xwpf.converter.pdf/src/test/resources/org/apache/poi/xwpf/converter/core
/TestTableCellAlignment.docx
```

Original comment by `angelo.z...@gmail.com` on 2 Nov 2012 at 3:11
- Changed state: **Fixed**
